third-party-podspecs/react-native-webview.podspec.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "react-native-webview",
+  "version": "11.6.2",
+  "summary": "React Native WebView component for iOS, Android, macOS, and Windows",
+  "license": "MIT",
+  "authors": {
+    "name": "Jamon Holmgren",
+    "email": "jamon@infinite.red"
+  },
+  "homepage": "https://github.com/react-native-webview/react-native-webview#readme",
+  "platforms": {
+    "ios": "9.0",
+    "osx": "10.13"
+  },
+  "source": {
+    "git": "https://github.com/react-native-webview/react-native-webview.git",
```

Comment:
We should update the forked repo to point to the WordPress mobile one.

Reply:
This was addressed in [this commit](https://github.com/wordpress-mobile/gutenberg-mobile/pull/3725/commits/9f45708eab965268f97a7a5d3465a23a7cb41562).